cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1113,9 +1113,12 @@ Slice::Gen::ForwardDeclVisitor::visitInterfaceDecl(const InterfaceDeclPtr& p)
 void
 Slice::Gen::ForwardDeclVisitor::visitEnum(const EnumPtr& p)
 {
+    string mappedName = p->mappedName();
+    string underlying = string("std::") + (p->maxValue() <= numeric_limits<uint8_t>::max() ? "uint8_t" : "int32_t");
```

Comment:
We should be consistent and use `::std::` everywhere.

Reply:
Fixed.